src/gui/src/layoutViewer.h

```diff
@@ -169,6 +169,9 @@ class LayoutViewer : public QWidget
   virtual void mouseMoveEvent(QMouseEvent* event) override;
   virtual void mouseReleaseEvent(QMouseEvent* event) override;
 
+  odb::Rect dbu_bounds;
```

Comment:
This will not update on resizes. I think it would be better to ask the window for its visible size instead of storing this value. (which tends not work correctly with Qt)

Reply:
I was looking for this option but didn't find how to do it, there is easy way to get the visible size from the window?

Reply:
you should change `getDBUBounds()` to return:
https://github.com/The-OpenROAD-Project/OpenROAD/blob/4066c4222535a116eb01adabc4e73ffc44c0d5ab/src/gui/src/layoutViewer.cpp#L3152

And in the process, maybe `getVisibleBounds()` is a better name for the function.

Reply:
Changed